analysis/interface/dmAnalysis.h

```diff
@@ -25,14 +23,11 @@ class dmAnalysis : public nanoBase {
   Bool_t b_trigger_dm;
 
 
-  pileUpTool* dm_pileUp;
-  lumiTool* dm_lumi;
-
   void MakeBranch(TTree* t);
   void resetBranch();
   bool analysis();
 
-  enum TTLLChannel { CH_NOLL = 0, CH_MUEL, CH_ELEL, CH_ZJets, CH_WJets };
+  enum TTLLChannel { CH_NOLL = 0, CH_MUEL, CH_ELEL, CH_MUMU, CH_ZJets, CH_WJets };
```

Comment:
You're not a ttbar dilepton analysis, so you shouldn't call your channels like that. Also, do you use all these different values?